l10n_eu_product_adr_report/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
+
+{
+    "name": "ADR Product report",
+    "summary": "",
+    "version": "11.0.1.0.0",
+    "depends": [
+        "product", "sale", "sales_team"
+    ],
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
+              "brain-tec AG,"
+              "Odoo Community Association (OCA)",
+    "category": "Product Management",
+    "website": "http://www.odoomrp.com",
+    'license': 'AGPL-3',
+    "data": [
+        "views/product_view.xml",
```

Comment:
Sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L30)